README.md

```diff
@@ -52,25 +52,67 @@ For 95% of use-cases of animating components, we don't have to resort to using h
 This library also provides an alternative, more powerful API for React's `TransitionGroup`.
 
 ## API
-The library exports `Motion`, `StaggeredMotion`, `TransitionMotion`, `presets`, `spring` and `utils`.
 
-#### `spring: number -> ?[stiffness, damping] -> ConfigurationObject`
-(**Note**: not the `Spring` component in version <0.3.0.)
-The pervasive helper used to specify the how to animate to the destination value, e.g. `spring(10, [120, 17])` returns an opaque configuration that describes "an animation to the value 10, with a physics spring's stiffness of 120 and damping of 17". `spring(10)` without the spring configuration array defaults to `[170, 26]`. See below for more usage and see [here](#presets) for a list of convenient configurations the library exports.
+### `type PhysicsConfig = [number, number]`
```

Comment:
Really not sure about this. If you check the final effect: https://github.com/bjz/react-motion/blob/restructure-readme/README.md#api it's a lot of ceremony before some actual docs